posix.mak

```diff
@@ -655,15 +659,19 @@ test: $(ASSERT_WRITELN_BIN)_test all
 # Changelog generation
 ################################################################################
 
+changelog/${NEXT_VERSION}_pending.dd: | ${STABLE_DMD} ../tools ../installer
+	$(STABLE_RDMD) $(TOOLS_DIR)/changed.d $(CHANGELOG_VERSION_MASTER) -o $@ \
+	--version "${NEXT_VERSION} (upcoming)" --date "To be released"
+
 changelog/${NEXT_VERSION}_pre.dd: | ${STABLE_DMD} ../tools ../installer
 	$(STABLE_RDMD) $(TOOLS_DIR)/changed.d $(CHANGELOG_VERSION_MASTER) -o $@ \
-	--version "${NEXT_VERSION} (upcoming)" --date "To be released" --nightly
+	--version "${NEXT_VERSION} (upcoming)" --date "To be released"
```

Comment:
The `changed` invocations for _pre and _pending look identical right now. Aren't they supposed to be for different things (master/stable)?